DigitalOffice/Services/ProjectService/Models/Project.cs

```diff
@@ -6,7 +6,7 @@ namespace LT.DigitalOffice.ProjectService.Models
     public class Project
     {
         public string Name { get; set; }
-        public IEnumerable<Guid> ManagersIds { get; set; }
-        public IEnumerable<Guid> WorkersIds { get; set; }
+        public List<Guid> ManagersIds { get; set; }
+        public List<Guid> WorkersIds { get; set; }
     }
```

Comment:
We removed field Managers, we added new fields for this in the db models.

Reply:
Apparently they forgot to fix it.

Reply:
Do we have such a rule? Can I have a proof?

Reply:
Fixed.